src/main/rx/jetiexbus.c

```diff
@@ -178,6 +185,14 @@ static void jetiExBusDataReceive(uint16_t c, void *data)
         }
     }
 
+    if (jetiExBusFramePosition == 1) {
+        if (c == 0x01) {
+            jetiExBusCanTx = true;
+        } else {
+            jetiExBusCanTx = false;
+        }
+    }
+
     if (jetiExBusFramePosition == jetiExBusFrameMaxSize) {
```

Comment:
THis test is not necessary - see `jetiExBusFrame[EXBUS_HEADER_MSG_LEN] <= EXBUS_MAX_CHANNEL_FRAME_SIZE)` later

Reply:
Removed

Reply:
That's really strange - I don't see any code path that would lead to overflow, at least in current code. 
Buffer should always hit EXBUS_HEADER_LEN, then jetiExBusFrameLength, but that is tested for overflow.

Only chance is that jetiExBusFrameState get (or remain) set to EXBUS_STATE_IN_PROGRESS, but jetiExBusFrame points to jetiExBusRequestFrame.

It's an interesting bug, I'll look on it soon.